### ISSUE TITLE
cantata: symbol lookup error

### ISSUE THREAD

**Issue by marrub--** [void-linux/void-packages#20627]
### System

* xuname:  
  `Void 5.4.29_1 x86_64 GenuineIntel notuptodate rrrmFFFF`
* package:  
  `cantata-2.4.1_1`

### Behaviour
When started, `cantata` crashes with:
```
cantata: symbol lookup error: /usr/lib/libneon.so.27: undefined symbol: SSL_CTX_set_post_handshake_auth
```
Presumably there's something wrong with the dependencies.

### Steps to reproduce the behavior
Install `cantata` and run it.

**Comment by Johnnynator** [comment]
This should have been fixed with d8656b4850ac9fda9460207bfbca3b1b3342c1e8 , can you please update `neon` and recheck if the issue persists.

**Comment by marrub--** [comment]
> This should have been fixed with [d8656b4](https://github.com/void-linux/void-packages/commit/d8656b4850ac9fda9460207bfbca3b1b3342c1e8) , can you please update `neon` and recheck if the issue persists.

Yep, that fixed it, thank you!
